core/server/server_ip.go

```diff
@@ -106,6 +145,22 @@ func runIPServer(log *zap.Logger, mtrcs *ipServerMetrics, conn *net.UDPConn, ifa
 
 		ntp.EncodePacket(&buf, &ntpresp)
 
+		if authenticated {
+			var cookies [][]byte
+			key := provider.Current()
+			for i := 0; i < len(ntsreq.Cookies)+len(ntsreq.CookiePlaceholders); i++ {
+				encryptedCookie, err := plaintextCookie.EncryptWithNonce(key.Value, key.Id)
+				if err != nil {
+					log.Info("failed to encrypt cookie", zap.Error(err))
+					continue
```

Comment:
Similar question as in `ntske.go`: Is it okay to just skip appending this cookie or do we actually need to abort from responding to this request altogether.

Reply:
Done